src/robot/output/console/dotted.py

```diff
@@ -79,7 +79,7 @@ def report(self, suite):
                               'test' if not suite.rpa else 'task',
                               plural_or_not(stats.total),
                               secs_to_timestr(suite.elapsedtime/1000.0)))
-        self._stream.highlight(suite.status + 'ED', suite.status)
+        self._stream.highlight(suite.status + ('P' if suite.status == 'SKIP' else '') + 'ED', suite.status)
```

Comment:
`'PED' if suite.status == 'SKIP' else 'ED'` could be a bit simpler construct here.